### DISCUSSION TITLE
Do I lose funds in channels if I lose phone?

### DISCUSSION THREAD

**Discussion by jbrepogmailcom** [ACINQ/phoenix#379] [Q&A]
Hello. I have read warning that during force-close I must not uninstall the app until all funds are settled.

Does this also apply not normal open funded channels? I think that LN channels are not restored from seed phrase, so what happens if I need to reinstall the application and restore wallet from seed? Are all funded channels lost? If yes, is there any way to backup LN channel state?

**Comment by Flowsmooth** [comment]
What funds do we get on GitHub am new to this

**Reply by Flowsmooth** [reply]
How do I know if I have funds please help me out with this please thank you

**Comment by dpad85** [comment]
>  during force-close I must not uninstall the app until all funds are settled.

That's because the force-close flow is special: there's a delay before the transaction sending funds back to your wallet is published. That's why you must not reset your wallet immediately, and why there's a warning. Regular channels closures (a.k.a mutual close) do not incur that delay.

> Does this also apply not normal open funded channels? I think that LN channels are not restored from seed phrase, so what happens if I need to reinstall the application and restore wallet from seed?

Channels are backed up by your peer. When you restore your wallet, Phoenix retrieves the encrypted backup and restores your channels. You can read more about this here: https://acinq.co/blog/introducing-phoenix-backup.
